test/e2e/tests/simple-send.spec.js

```diff
@@ -14,14 +19,28 @@ describe('Simple send', function () {
         ganacheOptions: defaultGanacheOptions,
         title: this.test.fullTitle(),
       },
-      async ({ driver, ganacheServer }) => {
-        await logInWithBalanceValidation(driver, ganacheServer);
+      async ({ driver }) => {
+        // Process
+        await unlockWallet(driver, WALLET_PASSWORD);
 
-        await sendTransaction(
-          driver,
+        // Page Objects and Components
+        const confirmTxPage = new ConfirmTxPage(driver);
+        const homePage = new HomePage(driver);
+        const sendAmountPage = new SendAmountPage(driver);
+        const sendToPage = new SendToPage(driver);
+
+        // Actions and Assertions
+        await homePage.startSendFlow();
+        await sendToPage.addRecipient(
           '0x985c30949c92df7a0bd42e0f3e3d539ece98db24',
-          '1',
         );
+        await sendAmountPage.addAmount('1');
```

Comment:
We want to keep page objects independent, so when we switch page, we don't return the object of the new class. To improve tests stability, i suggest we add waitPageLoaded() method to each class, so when we switch page in test spec, we can call it first to be sure that we are on the expected page and ready for actions.